src/task2.1/data-access/EntityRepository.js

```diff
@@ -0,0 +1,79 @@
+import { Op } from 'sequelize';
```

Comment:
This file looks a bit overkill. The appropriate logic may be described in service, so I don't see the reason to use additional file in data-access layer.